frame/support/test/tests/pallet_ui/store_trait_leak_private.stderr

```diff
@@ -1,3 +1,12 @@
+error: use of deprecated trait `pallet::Store`: Use of #[pallet::generate_store(pub(super) trait Store)] will be removed soon.
+               Check https://github.com/paritytech/substrate/pull/13535 for more details.
+  --> $DIR/store_trait_leak_private.rs:11:37
+   |
+11 |     #[pallet::generate_store(pub trait Store)]
+   |                                        ^^^^^
+   |
+   = note: `-D deprecated` implied by `-D warnings`
```

Comment:
This is an erroneous expectation update, please revert this and instead modify the test to not use `#[pallet::generate_store]`.